src/Compilers/Core/Portable/Generated/Operations.Generated.cs

```diff
@@ -2753,4 +2754,226 @@ internal interface IWithOperation : IOperation
         /// </summary>
         IOperation Value { get; }
     }
+    public abstract partial class OperationVisitor
+    {
+        public virtual void Visit(IOperation operation) => operation?.Accept(this);
+        public virtual void DefaultVisit(IOperation operation) { /* no-op */ }
+        internal virtual void VisitNoneOperation(IOperation operation) { /* no-op */ }
```

Comment:
place with the other internals? :D

Reply:
dumb question, why no-op?  The other internal visit methods don't do that.

Reply:
I'm going to leave it where it is for generation simplicity.

As to why a no-op, I honestly don't remember the reasoning. It's been that way since we shipped though, so I'm leaving it as is.